llvm/lib/Transforms/InstCombine/InstructionCombining.cpp

```diff
@@ -5034,6 +5034,105 @@ void InstCombinerImpl::tryToSinkInstructionDbgVariableRecords(
   }
 }
 
+void log_optzn(std::string Name) {
```

Comment:
I don't know what's going on but your pull request should not try to add this function, which is already present in the branch you're modifying. can you please make sure that your PR isn't trying to modify the main branch or something?